INSTALL-UBUNTU.md

```diff
@@ -0,0 +1,257 @@
+# Objective
+Install the FediBuzz relay using a virtual machine running Ubuntu Server 22.04.4 LTS. If you are familiar with NixOS/Flakes, then [installing the NixOS module](https://github.com/astro/buzzrelay?tab=readme-ov-file#build) for this is by far the easier route! 
```

Comment:
Mentioning the exact Ubuntu release will just make this guide look dated in no time. The steps should be generic enough that they very likely continue to work in the future.